client/src/telemetry/glean-context.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import { path, referrer } from "./generated/pageView";
+import { clicked } from "./generated/element";
+import * as pings from "./generated/pings";
+import * as Glean from "@mozilla/glean/web";
```

Comment:
Glean is the default export, so `import * as Glean` gives you an object with a key `default` and the actual `Glean` value as the value:

```suggestion
import Glean from "@mozilla/glean/web";
```